### ISSUE TITLE
Syntax highlighting for variable definition

### ISSUE THREAD

**Issue by Monokai** [SublimeText/Sass#99]
I think there's a general bug in colorization for variables in SCSS that include more than 1 whitespace character between the variable name and the colon character:

```
$color1: #f00;
$color2 : #f00;
$color3     : #f00;
```

<img width="209" alt="Screen Shot 2022-08-27 at 14 59 44" src="https://user-images.githubusercontent.com/277589/187031353-f2f9c34e-f9c7-4498-ae7c-0c4025e9ed99.png">

This seems to be the case for all variable types (strings, gradients, maps, etc.)

Use case: some people want multiple spaces before the colon character in order to visually line up the code:

```
$var-one        : #f00;
$var-twenty-one : #0f0;
$var-thirty     : #00f;
```

**Comment by braver** [comment]
I've got [a branch](https://github.com/braver/SublimeSass/tree/simplify) with a lot of rework to improve and simplify various cases. It should be fixed there now if you're interested in checking that out.

**Comment by Monokai** [comment]
Thanks. What's the best way of installing that branch in Sublime Text?

**Comment by braver** [comment]
I usually uninstall the package installed via package control, then check out the git repo into the Packages directory... something like this
```
git clone --branch simplify git@github.com:braver/SublimeSass.git "~/Library/Application Support/Sublime Text/Packages/Sass"
```

**Comment by Monokai** [comment]
Ah yes, that makes sense, I was looking for a way to install it via Package Control somehow.

And the new branch fixes this issue 👍

**Comment by braver** [comment]
Awesome. Still working out the final kinks, but an official release is coming. I believe package control will keep your checkout up to date with commits on that branch, so you're good for now 👌🏻

**Comment by braver** [comment]
closed via #96
